spec/SyntaxAggregator-spec.js

```diff
@@ -163,66 +166,170 @@ describe('SyntaxAggregator', () => {
+            concept_search: 'Variable Declaration in C#',
+            concept_permalink: 'variable-dec',
+            description:
+                'Used to declare a variable. Variables can be implicitly or explicitly typed.\r\n\r\nVariables declared this way (without a static modifier) within classes are called instance variables. They belong to an instance of the class (i.e. an object).',
+            syntax:
+                'modifier dataType variableName; ///modifier is optional\r\n\r\n///variables can be assigned values either separately or on declaration\r\nvariableName = value; ///separate line assignment\r\n\r\nmodifier dataType variableName = value; ///same line assignment\r\n\r\n///implicitly typed variable\r\nvar variable2 = value;',
+            notes:
+                'The modifier (public, private) permits or restricts direct access to the variable with respect to its scope (class, method, etc.). Variables declared within a class are called fields.\r\n\r\nVariables without a modifier are known as local variables, typically used within a method. They are temporary and only exist within the scope of the where its declared method.\r\n\r\ndataType is the data type of the variable. ',
+            example:
+                'public class Car { \r\n    private int speed; ///private variable declaration\r\n    public int wheels; ///public variable declaration\r\n    \r\n    /*...constructor, etc...*/\r\n\r\n    public void speedUp() {\r\n        ///local variable declaration, in line assignment, only seen within speedUp method\r\n        int speedIncrease = 10;\r\n        speed += speedIncrease;\r\n    }\r\n}',
+            keywords: 'fields',
+            related:
+                '<a href="/ref/csharp/access-mod">Access Modifiers</a>\r\n<a href="/ref/csharp/data-types">Primitive Data Types</a>',
+            documentation:
+                '<a href="https://msdn.microsoft.com/en-us/library/ms173104.aspx">Types (C# Programming Reference) - MSDN</a>',
+            language_permalink: 'csharp',
+        };
+
+        it('should display concept', () => {
+            expect(resultsPresenter.showResults).to.not.have.been.called;
+            syntaxAggregator.onSelect({
+                item: concept,
+                mode: LanguageFilterMode.SELECT_CONCEPT,
             });
+            expect(resultsPresenter.showResults).to.have.been.calledOnce;
+            expect(resultsPresenter.showResults).to.have.been.calledWith(
+                {
+                    item: concept,
+                    mode: LanguageFilterMode.SELECT_CONCEPT,
+                },
+                resultView,
+            );
         });
-        describe('list of concepts', () => {
-            it('should send list of concepts to the view', () => {
-                throw new Error('Not implemented');
+        it('should show panel for concept', () => {
+            // TODO move this to ResultsPresenter spec, since it tests whether presenter shows results panel
```

Comment:
Create `ResultsPresenter` spec, add this test case, and other test cases in that area to ensure that results are presented as expected.